packages/dev/serializers/test/integration/glTFSerializer.test.ts

```diff
@@ -555,5 +554,92 @@ describe("Babylon glTF Serializer", () => {
                 expect(node.mesh).toEqual(0);
             }
         });
+        it("should not export a root conversion node", async () => {
+            const assertionData = await page.evaluate(() => {
+                return BABYLON.SceneLoader.AppendAsync("https://models.babylonjs.com/Tests/TwoQuads/", "TwoQuads.gltf", window.scene).then((scene) => {
+                    scene.getMeshByName("__root__")!.name = "renamedRoot";
+                    return BABYLON.GLTF2Export.GLTFAsync(window.scene!, "test").then((glTFData) => {
+                        const jsonString = glTFData.glTFFiles["test.gltf"] as string;

```

Comment:
nit: `glTFFiles` is deprecated

```suggestion
                        const jsonString = glTFData.files["test.gltf"] as string;
```